src/components/input/bl-input.css

```diff
@@ -44,6 +45,10 @@
   --icon-size: var(--bl-font-body-text-3-line-height);
 }
 
+.wrapper:has(input:-webkit-autofill, input:-webkit-autofill:hover, input:-webkit-autofill:focus) {
+  box-shadow: 0 0 0 40rem var(--autofill-bg-color) inset;
```

Comment:
I understand the issue about overriding autofill style but since this is the wrapping element, you can just use `background-color`.